### DISCUSSION TITLE
Welcome to s25rttr-android Discussions!

### DISCUSSION THREAD

**Discussion by Farmer-Markus** [Farmer-Markus/s25rttr-android#2] [Announcements]
Write, ask anything you want to and feel free to share ideas.
I'll try to answer everything

**Comment by poborczykpiotr2-commits** [comment]
Help
<img width="576" height="1289" alt="Screenshot_20250929-074305" src="https://github.com/user-attachments/assets/07464fd6-d122-42da-bfb9-736771b25c98" />

**Reply by Farmer-Markus** [reply]
You need a copy of the original Settlers 2 Gold Edition game. Find and copy the "GFX" and "SND" Folder from the original game files to the folder you have chosen -> share -> s25rttr -> S2

There will be a file called "put your gamefiles here" (or something like this).
I you found this file, the same folder is where your gamefiles need to be placed

In your case the path should be: <internal storage>/FileManager/.s25rttr/share/s25rttr/S2

Maybe you need to "show hidden files" in your file explorer to see the .s25rttr directory (directories beginning with an . Are not shown in most file managers
